README.md

```diff
@@ -98,6 +98,16 @@ Install dependencies:
 $ npm install
 ```
 
+ Add SSL certificate:
+
+ The HTTPS server looks for privatekey.pem and certificate.pem. You can use a real certificate or generate a self-signed one by following the steps below.
+
+ ```
+ $ openssl genrsa -out privatekey.pem 2048
+ $ openssl req -new -sha256 -key privatekey.pem -out csr.pem
+ $ openssl x509 -req -in csr.pem -signkey privatekey.pem -out certificate.pem
+```
+
```

Comment:
Looks good to me. I added a script in the tools directory which contains those commands:
https://github.com/moziot/gateway/blob/master/tools/make-self-signed-cert.sh